controllers/subscriptions.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+
+import client from '@/services/GraphQL/client'
+import { CREATE_SUBSCRIPTION } from '@/services/GraphQL/subscriptions/mutations'
+
+export const createSubscription = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const { body } = req
+    const { data, errors: userErrors } = await client.mutate({
+      mutation: CREATE_SUBSCRIPTION,
+      variables: {
+        memberId: body.memberId,
+        expiration: body.expiration,
+      },
+    })
+
+    if (userErrors) {
+      return res.status(500).json(userErrors)
```

Comment:
Send a json in the form 

```js
{
msg: "Error while communicating with Data Base",
errors: userErrors
}
```

The idea is that the "Good" and the "Bad" response are similar in some way, and can be parsed easily by frontend like

```js
const { data, msg, errors } = async useRequest(...);

if (errors) console.log(msg, errors);
```